README.md

```diff
@@ -1,4 +1,7 @@
-#### About
+![Kodi logo](https://raw.githubusercontent.com/xbmc/xbmc-forum/master/xbmc/images/logo-sbs-black.png)
```

Comment:
Looks good for me.

For the logo you can use this way:
`[<img src="https://raw.githubusercontent.com/xbmc/xbmc-forum/master/xbmc/images/logo-sbs-black.png" alt="Home">](http://kodi.tv/)`

with them you have a link to kodi, instead the image.